### DISCUSSION TITLE
Skin Folders/Categories

### DISCUSSION THREAD

**Discussion by Zabrid-Chan** [ppy/osu#15626] [Ideas]
Some way to sort your skins into separate categories to make organization, and the ability to find a skin much easier. Basically meaning you can put your custom skins, dt skins, no mod skins, and other stuff in seperate folders/categories

Also being able to change skin names in-game would be cool

**Comment by peppy** [comment]
An initial step to make this work will be the ability to rename skins. Then you can make "folders" by adding a `Folder/` prefix to the skin name. You can expect this to happen as part of the next pieces of the skin editor (scheduled for some dev work once realm efforts are completed).
